openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -124,6 +124,9 @@ class Meta(object):
     marketing_url = TextField(null=True)
     eligible_for_financial_aid = BooleanField(default=True)
 
+    # Discussions
+    discussion_enabled = BooleanField(default=True)
```

Comment:
Probably doesn't matter much, but it should default to False.

Reply:
Thanks. This is done